File: C++/M01_ProgrammingBasics/L05_WhileLoop/Lab/Solutions/P05_AccountBalance.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string command;
	cin >> command;

	cout.setf(ios::fixed);
	cout.precision(2);

	double sum = 0.0;
	while (command != "NoMoreMoney") {
		double currValue = stod(command);

		if (currValue < 0) {
			cout << "Invalid operation!" << endl;
			break;
		}

		sum += currValue;
		cout << "Increase: " << currValue << endl;

		cin >> command;
	}

	cout << "Total: " << sum << endl;
	
	return 0;
}

File: Python/M01_ProgrammingBasics/L05_WhileLoop/Lab/Solutions/P05_AccountBalance.py
total = 0
line = input()

while line != "NoMoreMoney":
    current = float(line)

    if current < 0:
        print("Invalid operation!")
        break
    total += current
    print(f"Increase: {current:.2f}")

    line = input()

print(f"Total: {total:.2f}")
